src/opcodes/mod.rs

```diff
@@ -2370,10 +2375,401 @@ mod test {
             accumulator: 0x00,
             ..cpu_initial
         };
-        cpu_expected.set_flag(StatusFlag::Zero, true);
+        cpu_expected.set_flag(StatusFlag::Zero, false);
+        cpu_expected.set_flag(StatusFlag::Negative, true);
+
+        adc(&mut cpu_initial, &mut stub_bus, AddressModeValue::AbsoluteAddress(0x00ff));
+
+        assert_eq!(cpu_initial, cpu_expected);
+    }
+
+    #[test]
+    #[cfg(feature = "binary_coded_decimal")]
+    fn test_adc_decimal_1() {
+        const input: (u8, u8, bool) = (0x00, 0x00, false);
+        const expected: (u8, bool, bool, bool, bool) = (0x00, false, false, true, false);
```

Comment:
Small linting thing, constants should be named in all caps. Same goes for the other tests.
```suggestion
        const INPUT: (u8, u8, bool) = (0x00, 0x00, false);
        const EXPECTED: (u8, bool, bool, bool, bool) = (0x00, false, false, true, false);
```